openslides_backend/services/media/interface.py

```diff
@@ -13,3 +13,6 @@ def upload(
         mimetype: str,
     ) -> None:
         ...
+
+    def upload_resource(self, file: str, id: int, mimetype: str) -> str:
+        ...
```

Comment:
Did you not implement this on purpose? If so, please open an issue for it, or, better, directly implement it. It will lead to errors in the full setup and is especially confusing since the tests all pass.

Reply:
Implemented and changed for mediafile